test/evergreen/compatibility_test_for_releases.sh

```diff
@@ -262,13 +370,13 @@ fi
 
 if [ "$older" = true ]; then
     for b in ${older_release_branches[@]}; do
-        (run_format $b "fix row var")
+        (run_format $b "row")
```

Comment:
This feels like testing coverage shrink for older release branches. Did we run into any issue for column stores on those branches?

Reply:
Reverted the change in older release branches.